Exercise_nonsequential/array_job_nonsequence.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# This script runs the array job spawned using the main.sh script on a
+# single node on express partition to process folders starting with
+# name sub-blast located inside a work directory.
+
+#SBATCH -J slurm_python
+#SBATCH --partition=reservation                                                                                                                                                                                    
+#SBATCH --reservation=fall_training_cpu_2022
+#SBATCH --partition=express
+#SBATCH -o %A_%a.o 
+#SBATCH -e %A_%a.e 
+#SBATCH --nodes=1
+#SBATCH --ntasks-per-node=1
+#SBATCH --cpus-per-task=1
+#SBATCH --time=00:20:00
+#SBATCH --mem=4G 
+
+# Use a regular partition, such as express, when not in a training                                                                                                                                                                  
+# session                                                                                                                                                                                                                   
+##SBATCH --partition=express
```

Comment:
This comment is superfluous and can be removed.

Reply:
@dhml , thanks for your comments. We typically keep the comment about using regular partition outside of the training session because in the past we've had tickets from attendees of the session about the partition not working out from them because they were continuing to use the reservation partition.

Reply:
@manasvita, understood.  Perhaps remove this and move the textual comment to `#SBATCH --partition=...` above?

Reply:
@dhml , since these scripts will be used during training tomorrow I have the #SBATCH --partition=reservation and it's corresponding name in the main SBATCH directive. Not sure, if I follow what you mean by 
"#SBATCH --partition=... above?"

Reply:
@manasvita, I meant something like the following:
```
...
#SBATCH -J slurm_python
# Use a regular partition such as "express" when not in a training session
#SBATCH --partition=reservation
#SBATCH --reservation=fall_training_cpu_2022
...
```

Reply:
@dhml , I see. Thanks for the clarification. That could work too and I can just add a comment to remove --reservation flag as well when not in training.